GitUI/CommandsDialogs/FormCleanupRepository.Designer.cs

```diff
@@ -28,231 +28,299 @@ protected override void Dispose(bool disposing)
+            checkBoxIncludePathFilter.CheckedChanged += checkBoxPathFilter_CheckedChanged;
+            // 
+            // textBoxExcludePaths
+            // 
+            textBoxExcludePaths.Anchor = AnchorStyles.Top | AnchorStyles.Left | AnchorStyles.Right;
+            textBoxExcludePaths.Location = new Point(60, 362);
+            textBoxExcludePaths.Margin = new Padding(4);
+            textBoxExcludePaths.Multiline = true;
+            textBoxExcludePaths.Name = "textBoxExcludePaths";
+            textBoxExcludePaths.ScrollBars = ScrollBars.Vertical;
+            textBoxExcludePaths.Size = new Size(466, 78);
+            textBoxExcludePaths.TabIndex = 10;
+            // 
+            // checkBoxExcludePathFilter
+            // 
+            checkBoxExcludePathFilter.AutoSize = true;
+            checkBoxExcludePathFilter.Location = new Point(24, 331);
+            checkBoxExcludePathFilter.Margin = new Padding(4);
+            checkBoxExcludePathFilter.Name = "checkBoxExcludePathFilter";
+            checkBoxExcludePathFilter.Size = new Size(252, 24);
+            checkBoxExcludePathFilter.TabIndex = 8;
+            checkBoxExcludePathFilter.Text = "Exclude the following file path(s):";
+            checkBoxExcludePathFilter.UseVisualStyleBackColor = true;
+            checkBoxExcludePathFilter.CheckedChanged += checkBoxExcludePathFilter_CheckedChanged;
+            // 
+            // AddExclusivePath
+            // 
+            AddExclusivePath.Anchor = AnchorStyles.Top | AnchorStyles.Right;
+            AddExclusivePath.ImageAlign = ContentAlignment.MiddleLeft;
+            AddExclusivePath.Location = new Point(378, 325);
+            AddExclusivePath.Margin = new Padding(4);
+            AddExclusivePath.Name = "AddExclusivePath";
+            AddExclusivePath.Size = new Size(150, 31);
+            AddExclusivePath.TabIndex = 9;
+            AddExclusivePath.Text = "Add a path...";
+            AddExclusivePath.UseVisualStyleBackColor = true;
+            AddExclusivePath.Click += AddExcludePath_Click;
+            // 
+            // labelPathHintExclude
+            // 
+            labelPathHintExclude.AutoSize = true;
+            labelPathHintExclude.Location = new Point(62, 446);
+            labelPathHintExclude.Margin = new Padding(4, 0, 4, 0);
+            labelPathHintExclude.Name = "labelPathHintExclude";
+            labelPathHintExclude.Size = new Size(132, 20);
+            labelPathHintExclude.TabIndex = 11;
+            labelPathHintExclude.Text = "(one path per line)";
+            // 
+            // labelPathHintInclude
+            // 
+            labelPathHintInclude.AutoSize = true;
+            labelPathHintInclude.Location = new Point(62, 294);
+            labelPathHintInclude.Margin = new Padding(4, 0, 4, 0);
+            labelPathHintInclude.Name = "labelPathHintInclude";
+            labelPathHintInclude.Size = new Size(132, 20);
+            labelPathHintInclude.TabIndex = 12;
```

Comment:
Can you please verify and confirm the tab order is logical across the whole dialog?

Reply:
Updated the tab order, I'm not positive on best practice in this regard but I think it's good now.